.github/workflows/just-run-tests.yml

```diff
@@ -29,6 +29,19 @@ on:
         default: true
         type: boolean
         description: Run e2e tests (assessment)
+      run_python_e2e_tests:
+        required: false
+        default: true
+        type: boolean
+        description: Whether to run the python e2e tests
+      python_e2e_copilot_environment:
+        description: "Copilot environment to run tests on"
+        type: choice
+        options:
+          - dev
+          - test
```

Comment:
This seems to overlap with the `dev`/`test`/`uat` flags - should we just reuse those rather than add another one?

Reply:
@samuelhwilliams Have implemented this in c0dc1698aec8f2f1662699c6883dcc504fa7af4b